### PR TITLE
Added shorthand method for updating field and column labels

### PR DESCRIPTION
**Pull request by FrittenKeeZ**
Just making it easier to only update labels for fields and columns set from database.

### CODE REVIEW

File: src/PanelTraits/Columns.php [hunk 1]
```diff
@@ -196,6 +196,17 @@ public function setColumnDetails($column, $attributes)
     }
 
     /**
+     * Set label for a specific column.
+     *
+     * @param string $column
+     * @param string $label
+     */
+    public function setColumnLabel($column, $label)
```
Comment (lloy0076) [inline src/PanelTraits/Columns.php]: If they're string up there, may as well type hint them as string down here?

File: src/PanelTraits/Fields.php [hunk 2]
```diff
@@ -154,6 +154,22 @@ public function removeFields($array_of_names, $form = 'both')
     }
 
     /**
+     * Set label for a specific field.
+     *
+     * @param string $field
+     * @param string $label
+     */
+    public function setFieldLabel($field, $label)
```
Comment (lloy0076) [inline src/PanelTraits/Fields.php]: Maybe type hint there?

**Comment by FrittenKeeZ** [comment]
Yeah I thought about the third parameter (`update/create/both`) for fields, but left it out due to the same reasoning, but it wouldn't hurt to add.
In regards to type hinting, that would force the code to be PHP 7+ only.

File: src/PanelTraits/Fields.php (hunk 2 above)
Comment (lloy0076) [inline src/PanelTraits/Fields.php]: Oh confound PHP 5.6 compatibility - I forgot about that 👎

**Comment by FrittenKeeZ** [comment]
@tabacitu any chance you could have a look at this?

**Comment by tabacitu** [comment]
Very nice and clean. I like the idea - and flawless implementation. Thank you @FrittenKeeZ . Merging it into ```dev``` right now, it will be available on master later this week.

Cheers!

**Pull request merged**